A14-channelz.md

```diff
@@ -728,8 +730,27 @@ message ChannelData {
   google.protobuf.Timestamp last_call_started_timestamp = 7;
 }
 
+message ChannelTraceEvent {
```

Comment:
Also here for adding a comment.   Protos are the defacto spec and documentation, so they are the source of truth.   Each message and field should have a comment usually.  (I may have not be as disciplined in my messages, but I should have been!)

Reply:
Done